crates/bevy_ecs/src/world/mod.rs

```diff
@@ -946,8 +962,8 @@ impl World {
         match self.get_resource() {
             Some(x) => x,
             None => panic!(
-                "Requested resource {} does not exist in the `World`. 
```

Comment:
I'm unsure if removing trailing whitespace will do what we want. Either way, this is noise in a tricky PR: can you please remove this set of changes?

Reply:
Yup, I didn't notice this get in here and I have my editor set to do this automatically.